src/components/interfaces/HMI_API.xml

```diff
@@ -5663,6 +5678,33 @@
   </function>
   <function name="SendHapticData" messagetype="response">
   </function>
+
+  <function name="OnUpdateFile" messagetype="notification">
```

Comment:
the proposal obviously meant notification, correct? we don't need to modify where the proposal says request?

Reply:
Yes it should be notification.